resources/views/dashboard/training/tickets.blade.php

```diff
@@ -107,9 +107,9 @@
                         @if($tickets->count() > 0)
                             @foreach($tickets as $t)
                                 @if($t->sort_category == $trainingCategory)
-                                    @if($t->cert) {{-- student certified: green highlight --}}
+                                    @if($t->cert)
                                         <tr class="table-success">
-                                    @elseif($t->monitor) {{-- student may be monitored: blue highlight --}}
```

Comment:
I know you prefer no comments in the code and I trust your opinion. I do think it is more difficult to understand the function of vaguely defined controls like this.

Reply:
I'll look at this and see if I can rewrite the code slightly to be more descriptive so the comment isn't necessary. If that becomes too time consuming, I'll leave the comment for now

Reply:
@kjporter I went ahead and added these comments back for now. We can look into this more at another time.